openapi.yml

```diff
@@ -216,7 +216,7 @@ paths:
       requestBody:
         required: true
         content:
-          application/x-www-form-urlencoded:
+          application/json:
```

Comment:
@mjgiarlo !!!!

Reply:
@ndushay yessss? This was another instance where the prescat test suite tested different behavior from the implementation in pres-client, if this is like the other related change. I trusted the test suite. 😁